core/src/components/slider/slider.ts

```diff
@@ -623,9 +685,83 @@ export function createSlider(config?: PropsConfig<SliderProps>): SliderWidget {
 		patch,
 		api: {},
 		directives: {
-			sliderDirective: mergeDirectives(sliderDirective, resizeDirective),
-			minLabelDirective,
-			maxLabelDirective,
+			sliderDirective: mergeDirectives(sliderDirective, resizeDirective, containerDirective),
+			progressDisplayDirective: createAttributesDirective((progressContext$: ReadableSignal<{option: ProgressDisplayOptions}>) => ({
+				styles: {
+					left: computed(() => progressContext$().option.left + '%'),
+					right: computed(() => progressContext$().option.right + '%'),
+					top: computed(() => progressContext$().option.top + '%'),
+					bottom: computed(() => progressContext$().option.bottom + '%'),
+					width: computed(() => progressContext$().option.width + '%'),
+					height: computed(() => progressContext$().option.height + '%'),
+				},
+				classNames: {
+					'au-slider-progress': true,
+				},
+			})),
+			clickableAreaDirective: createAttributesDirective(() => ({
+				events: {
+					click: (event: MouseEvent) => widget.actions.click(event),
+				},
+				classNames: {
+					'au-slider-clickable-area': horizontal$,
+					'au-slider-clickable-area-vertical': vertical$,
+				},
+			})),
+			handleDirective: createAttributesDirective((handleContext$: ReadableSignal<{item: SliderHandle}>) => ({
+				events: {
+					keydown: (event: KeyboardEvent) => widget.actions.keydown(event, handleContext$().item.id),
+					mousedown: (event: MouseEvent) => widget.actions.mouseDown(event, handleContext$().item.id),
+					touchstart: (event: TouchEvent) => widget.actions.touchStart(event, handleContext$().item.id),
+				},
+				attributes: {
+					role: 'slider',
+					'aria-valuemin': min$,
+					'aria-valuemax': max$,
+					'aria-valuenow': computed(() => handleContext$().item.value),
+					'aria-valuetext': computed(() => handleContext$().item.ariaValueText),
+					'aria-label': computed(() => handleContext$().item.ariaLabel),
+					'aria-orientation': computed(() => (vertical$() ? 'vertical' : undefined)),
+					'aria-disabled': computed(() => (disabled$() ? 'true' : undefined)),
+					disabled: computed(() => (disabled$() ? 'true' : undefined)),
```

Comment:
This must be `disabled: disabled$`. It is not like aria-disabled where "true" (as a string) is expected. This should explain the change in bootstrap-slider-default.html.

```suggestion
					disabled: disabled$,
```